package.json

```diff
@@ -5,9 +5,10 @@
   "main": "dist/index.js",
   "scripts": {
     "prepublish": "npm run build",
-    "build": "mkdir -p dist && babel src --out-dir dist",
-    "test": "mocha test",
-    "lint": "eslint ./src"
+    "build": "NODE_ENV=production babel --out-dir dist src",
```

Comment:
why was this changed? Best to keep this PR related to react 16 compat only

Reply:
Using `babel-preset-react-app` requires that you specify `NODE_ENV`